### DISCUSSION TITLE
Kaskadierte Wechselrichter

### DISCUSSION THREAD

**Discussion by g5porthe** [bolliy/ioBroker.sun2000#144] [Q&A] [answered]
Ich habe drei Wechselrichter zusammengeschaltet. Zwei davon haben auch eine Batterie. Über den Adapter sehe ich nur den "Master" wo auch der SDongle drauf steckt. Gibt es eine Möglichkeit auch auf die Werte der anderen WR zuzugreifen?

**Comment by bolliy** [comment, marked as answer]
In der Konfiguration des sun2000 Adapters im "Main tab" die Slave WR zusätzlich durch ein Komma getrennt eintragen.

  
**modbusIds**: master, slave 1, slave 2

Siehe: 
https://github.com/bolliy/ioBroker.sun2000/blob/main/docs/configuration.md

Stephan

**Comment by g5porthe** [comment]
Vielen Dank!
Endlich sehe ich alle Daten.
BG Gerhard

Am So., 29. Dez. 2024 um 20:53 Uhr schrieb bolliy ***@***.***
